odoo/addons/base/wizard/base_module_uninstall.py

```diff
@@ -9,25 +9,25 @@ class BaseModuleUninstall(models.TransientModel):
     _description = "Module Uninstall"
 
     show_all = fields.Boolean()
-    module_id = fields.Many2one(
+    module_ids = fields.Many2many(
         'ir.module.module', string="Module", required=True,
         domain=[('state', 'in', ['installed', 'to upgrade', 'to install'])],
         ondelete='cascade', readonly=True,
     )
-    module_ids = fields.Many2many('ir.module.module', string="Impacted modules",
+    impacted_module_ids = fields.Many2many('ir.module.module', string="Impacted modules",
                                   compute='_compute_module_ids')
     model_ids = fields.Many2many('ir.model', string="Impacted data models",
                                  compute='_compute_model_ids')
 
     def _get_modules(self):
         """ Return all the modules impacted by self. """
-        return self.module_id.downstream_dependencies(self.module_id)
+        return self.module_ids.downstream_dependencies(self.module_ids)
 
-    @api.depends('module_id', 'show_all')
+    @api.depends('module_ids', 'show_all')
     def _compute_module_ids(self):
```

Comment:
For the sake of consistency, please rename the method to `_compute_impacted_module_ids`.